.travis.yml

```
@@ -3,10 +3,13 @@ julia:
   - 1.1
   - nightly
 
+cache: ccache
+
 env:
   global:
     - MAKEFLAGS="-j4"
     - JULIA_TRACK_COVERAGE=yes
+    - PATH="/usr/lib/ccache:$PATH"
```

This is not necessary, travis adds `/usr/lib/ccache` automatically to the path when ccache is enabled.
```suggestion
    - PATH="/usr/lib/ccache:$PATH"
```

---

As I understood it, it only does so with `language: c` or `language: c++`. In any case, I don't see `/usr/lib/ccache` twice in the PATH... But sure, I can try what happens if I remove that line.

---

Ah yeah, that might be a restriction. No, it is fine for me as it is.